src/wallet/mod.rs

```diff
@@ -74,10 +79,10 @@ impl WalletManager {
     /// An internal method for loading the wallet storage.
     ///
     /// TODO: There should be a file lock around this.
-    async fn load_unlocked(&self) -> anyhow::Result<WalletStorage> {
+    pub(crate) async fn load_unlocked(&self) -> anyhow::Result<WalletStorage> {
         let file = File::open(&self.path)
             .await
-            .with_context(|| format!("Opening {:?}", &self.path))?;
+            .with_context(|| format!("Opening {:?}", &self.path)).unwrap(); // we want the program to crash with error if the file doesn't exist
```

Comment:
I think we want it to chain the errors up the call stack, so there's a trace of all the different contexts.

Remember this could be used in a library too.

Reply:
Actually feeless will be using it as a library with the RPC server!